### PR TITLE
Refactor cropper

### PR DESCRIPTION
**Pull request by RemcoSimonides**
State issue number in your PR to close it when PR merged: issue #XXX
e.g. : "Resolve the bundle size ❤️: issue #1234"

### CODE REVIEW

File: libs/media/src/lib/file/file-new-uploader/file-uploader.component.ts [hunk 1]
```diff
@@ -178,4 +178,20 @@ export class FileUploaderComponent implements OnInit {
       }
     }
   }
+
+  private computeState() {
```
Comment (RemcoSimonides) [inline libs/media/src/lib/file/file-new-uploader/file-uploader.component.ts:182]: Actually double in here now... removing it

File: libs/movie/src/lib/movie/components/picture-upload/picture-upload.component.ts [hunk 2]
```diff
@@ -27,7 +26,7 @@ export class MoviePictureUploadComponent implements OnInit {
   }
 
   addStill() {
-    this.moviePictureForm.get('still_photo').push(new HostedMediaForm());
+    this.moviePictureForm.get('still_photo').push({ storagePath: '' });
```
Comment (RemcoSimonides) [inline libs/movie/src/lib/movie/components/picture-upload/picture-upload.component.ts:29]: Not sure if `{ storagePath: '' }` is needed. Could just be `''`?

File: libs/movie/src/lib/movie/form/media-files/media-files.component.ts [hunk 3]
```diff
@@ -20,6 +20,21 @@ export class MovieFormMediaFilesComponent {
...
+      metadata: {
+        privacy: 'public',
+        collection: 'movies',
+        docId: this.movieId,
+        field: 'promotional.presentation_deck.storagePath',
+        uid: '',
+      }
+    }
+  }
+
```
Comment (RemcoSimonides) [inline libs/movie/src/lib/movie/form/media-files/media-files.component.ts:37]: Removing this too

File: libs/movie/src/lib/movie/components/picture-upload/picture-upload.component.ts (hunk 2 above)
Comment (pldespaigne) [reply libs/movie/src/lib/movie/components/picture-upload/picture-upload.component.ts:29]: I think it should be `push(new StorageFileForm())`

**Review by pldespaigne — CHANGES_REQUESTED** [review]

**Review by pldespaigne — APPROVED** [review]

**Pull request merged**